presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,93 @@
+================================
+File Based System Access Control
```

Comment:
There is another section in prestodb docs with the same header "10.6. System Access Control". So I left the header as it is, since didn't want to have two headers with the same name. We can think if we can somehow consolidate common parts of the two sections.